packages/app/src/core/constants/state.constant.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export enum State {
+  Initial = 0,
```

Comment:
Is the assignment needed?

Reply:
https://github.com/mudita/mudita-center/pull/914/commits/b57e8f74e6f7294627c128c37a4d33837de9dd8f